easybuild/easyblocks/s/scotch.py

```diff
@@ -108,7 +117,7 @@ def build_step(self):
         if self.toolchain.options['i8']:
             cflags += " -DINTSIZE64"
 
-        if not self.toolchain.mpi_family() in [toolchain.INTELMPI, toolchain.QLOGICMPI]:  #@UndefinedVariable
+        if self.cfg['threadedmpi']: 
```

Comment:
@ocaisa you could use `None` as default value, and assume people will set it to `False` in the easyconfig?

and then check `self.cfg['threadedmpi'] is None`

there's no easy way to distinguish between a default value and the same value but specified by the easyconfig